code42.json

```diff
@@ -629,6 +629,242 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+                {
+            "action": "create user",
+            "identifier": "create_user",
+            "description": "Create a new Code42 user account",
+            "type": "contain",
+            "read_only": false,
+            "parameters": {
+                "org_uid": {
+                    "description":  "Org UID to create user in",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "org_id"
+                    ],
+                    "default": "",
+                    "order": 0
+                },
+                "username": {
+                    "description":  "Username to create",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "default": "",
+                    "order": 1
+                },
+                "password": {
+                    "description":  "Password for newly created user (defaults to randomly generated password, requiring reset before login)",
+                    "data_type": "string",
```

Comment:
Should the data type for this parameter be `password`? I believe that setting it as such will do things like encrypt it on Phantom's backend and obfuscate the values when entered in the UI